compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -3987,6 +3987,7 @@ private boolean isInTypeDescContext() {
             case TYPE_DESC_IN_SERVICE:
             case TYPE_DESC_IN_PATH_PARAM:
             case STMT_START_BRACKETED_LIST:
+            case DEFAULTABLE_PARAM:
```

Comment:
DEFAULTABLE_PARAM is not a part of the type-descriptor. Defining it like this is wrong.

Reply:
Fixed.